autotest/cpp/test_viewshed.cpp

```diff
@@ -62,21 +62,19 @@ Viewshed::Options stdOptions(const Coord &observer)
     return stdOptions(observer.first, observer.second);
 }
 
-DatasetPtr runViewshed(int8_t *in, int edgeLength,
-                       const Viewshed::Options &opts)
+DatasetPtr runViewshed(const int8_t *in, int xlen, int ylen,
+                       Viewshed::Options opts)
```

Comment:
any reason to have back out the change I did in master? (Coverity Scan complains when too large structures are passed by values)
```suggestion
                       const Viewshed::Options &opts)
```

Reply:
No. I didn't realize you had made changes.